lib/global_id_utils/locator.rb

```diff
@@ -39,10 +46,5 @@ def self.unscoped(model_class)
       end
     end
     private_class_method :unscoped
-
-    def self.find(model_class, id)
-      model_class.find_by(model_class.gid_model_id_attribute => id)
```

Comment:
I mistakenly have always thought of `#find_by` as "the same" as `#find` except with a given column. That turns out to not be the case. As noted in the PR description, `#find_by` lacks two qualities of `#find`:

1. It will only return one record even if you give it an array of IDs
2. It will not raise an error if no records are found

Anyway, TIL.